chain-signatures/node/src/metrics.rs

```diff
@@ -51,6 +51,26 @@ pub(crate) static SIGN_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
     .unwrap()
 });
 
+pub(crate) static SIGN_GENERATION_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "multichain_sign_gen_latency_sec",
+        "Latency of multichain signing, from received sign request to signature generation complete.",
```

Comment:
Is it from receiving or from starting the protocol?

Reply:
start of the protocol

Reply:
Let's modify the description then, these metrics are becoming hard to understand

Reply:
done